pcsx2/USB/usb-pad/usb-pad.h

```diff
@@ -27,6 +27,14 @@ namespace usb_pad
 		CID_STEERING_R,
 		CID_THROTTLE,
 		CID_BRAKE,
+		CID_DJ_CROSSFADER_LEFT,
```

Comment:
Please try to re-use the existing enumeration values, rather than adding new values.

Reply:
yeah i can do that, just wasn't sure what the recommendation was there

Reply:
I didn't really see any examples of other pad types that used any other axis' besides steering wheel related stuff though.

Reply:
You can reuse the sterring/throttle/etc axes for anything else.

Reply:
But then i would be relying on the logic in GetBindValue for that id right?

Reply:
Hows that? I just used some of the button CIDs, as there weren't going to be enough steering ones.

Reply:
In that case, it may be better to create a completely separate device, rather than piggybacking off the wheels.

Reply:
Ah yeah, good call, ill do that

Reply:
Can pretty much just copy+rename usb_pad, and strip all the bits you don't need.

I know it's extra work, but it just feels a little wrong to add all these fields which are only used by one controller type. Tbh, half of the controllers in there already aren't wheels, but I didn't have the energy to duplicate them when I rewrote all the USB stuff last year.

Reply:
Yeah, that is perfectly valid, i have done this now :). Also means i could leave a stub there for handling the euphoria led, in case someone wants to handle that on xinput using vibration commands, or if we ever get real PS3 turntable handling then it leaves it open for that too.